src/carbon/email.cr

```diff
@@ -45,7 +45,6 @@ abstract class Carbon::Email
     # layout email_layout
     # ```
     macro layout(template_name)
-      # Use the layout in `templates/\{{ template_name.id }}/layout.ecr`
```

Comment:
This is fixed in Crystal 1.5.0 https://github.com/crystal-lang/crystal/pull/12175